### ISSUE TITLE
Any period at the start of a line highlights as a scene heading

### ISSUE THREAD

**Issue by DaTedinator** [superlou/fountain-atom#94]
Fountain forces a scene heading with a single period followed by an alphanumeric character, but right now the package is highlighting any line that starts with a period as a scene heading. In my instance specifically, it's interpreting ellipses as a heading, when they should just be another action line.

Sample script:
```
EXT. QUARRY

Paul and Steve throw rocks at each other.

PAUL
Here's a fast one!

He throws a rock as hard as he can...

...but misses by a mile.

STEVE
Missed me!
```
![image](https://user-images.githubusercontent.com/29130403/71637564-c8dd8300-2bfa-11ea-8011-534b96517ca3.png)

Hope this one isn't as annoying to fix as my other issue.

**Comment by superlou** [comment]
The latest commit to master should only create scene headings for a single leading period now. Can you pull the latest and run in development mode to see if that handles it before I push a new patch? If you can mess around and see if there are any other funny cases for scene headings, I'd really appreciate it since we don't have great test coverage on the syntax grammar.

**Comment by DaTedinator** [comment]
Yeah, it seems like that fixed it. I played around a bit but can't think of a lot of other scenarios that sort of thing might come up; it seems good to me.

**Comment by superlou** [comment]
Ok. Published in 0.8.3.
